cinema-soft/src/main/java/softserve/academy/cinemasoft/repository/RoleRepository.java

```diff
@@ -1,11 +1,10 @@
 package softserve.academy.cinemasoft.repository;
 
-import softserve.academy.cinemasoft.model.Role;
-
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
+import softserve.academy.cinemasoft.model.Role;
 
 @Repository
-public interface RoleRepository extends JpaRepository<Role,Integer> {
+public interface RoleRepository extends JpaRepository<Role, Integer> {
     Role findByName(String name);
```

Comment:
Separate the methods from the class definition with an empty line for readability.

Reply:
Ok